cloud/pkg/admissioncontroller/constants/default.go

```diff
@@ -0,0 +1,9 @@
+package constants
+
+const (
+	ExternalAdmissionWebhookName = "deny-unwanted-devicemodel.k8s.io"
```

Comment:
will it be a good idea to rename to ``` validation-devices.kubeedge.io ``` ? devices.kubeedge.io is the group for both devices and devicemodels and we can use this as a framework and add more validation cases. Thoughts @chendave @kevin-wangzefeng @fisherxu @rohitsardesai83

Reply:
admission control should not limited with device/devicemodel, definitely, this will be renamed and whole thing should be refactor to make it more compatible and flexible. 

I am good with change this to `validation-devices.kubeedge.io` for now.